Fabric.Terminology.API/web.config

```diff
@@ -5,6 +5,15 @@
   For more info see https://go.microsoft.com/fwlink/?linkid=838655 -->
 
   <system.webServer>
+    <!--
+    TODO being discussed in PR review.
```

Comment:
Are we supposed to discuss this? I think this is a good change

Reply:
Nope - just want to make a note to make sure I cleaned it up.  I'll remove it and merge.